test/functional/api/arbmark_spec.lua

```diff
@@ -0,0 +1,18 @@
+-- Sanity checks for buffer_* API calls via msgpack-rpc
```

Comment:
Moving `viml/*` to `eval/` is fine with me. I think they were created by coincidence long ago.